README.md

```diff
@@ -1,8 +1,12 @@
 # joinmarket-clientserver
 
-Joinmarket refactored to separate client and backend operations
+JoinMarket is software to create a special kind of bitcoin transaction called a CoinJoin transaction. It's aim is to improve the confidentiality and privacy of bitcoin transactions.
 
-Joinmarket is coinjoin software, and includes a wallet, which requires Bitcoin Core as backend (version 0.16.3, 0.17.0 or later).
+A CoinJoin transaction requires other people to take part. The right resources (coins) have to be in the right place, at the right time, in the right quantity. This isn't a software or tech problem, its an economic problem. JoinMarket works by creating a new kind of market that would allocate these resources in the best way.
```

Comment:
I'm ambivalent about adding too much text to the README before the action items. But on reflection this text is probably needed here.
I did like adding the bullet point list because I think people don't know the list of features.
Btw you could add a few words mentioning auto-freezing to the coin control entry in the bullet points, too, now that's added.